doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+
+Feature comparison
+------------------
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
+
+    *   -   Parameter
+        -   `igorcoding/asynctnt <https://github.com/igorcoding/asynctnt>`__
+        -   `shveenkov/aiotarantool <https://github.com/shveenkov/aiotarantool>`__
+        -   `shveenkov/gtarantool <https://github.com/shveenkov/gtarantool>`__
+        -   `tarantool/tarantool-python <https://github.com/tarantool/tarantool-python>`__
+
+    *   -   License
+        -   Apache License 2.0
+        -   LGPL
+        -   LGPL
+        -   Modified BSD
+
+    *   -   Is maintained
+        -   Yes
+        -   No (last updated in 2016)
+        -   No (last updated in 2018)
+        -   Yes
+
+    *   -   Estimation of importance of known bugs
+        -   `issue #18 <https://github.com/igorcoding/asynctnt/issues/18>`__
+        -   `issue #5 <https://github.com/shveenkov/aiotarantool/issues/5>`__
+            (select() fails), `issue #16
+            <https://github.com/shveenkov/aiotarantool/issues/16>`__ (Tarantool
+            1.7 is unsupported)
+        -   No known bugs
+        -   `issue #105 <https://github.com/tarantool/tarantool-python/issues/105>`__ (Unpack of binary data)
+
+    *   -   Documentation
+        -   Yes (`github.io <https://igorcoding.github.io/asynctnt/>`__)
+        -   No
+        -   No
+        -   Yes (`tarantool.io
+            <https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/>`__
+            and `readthedocs
+            <https://tarantool-python.readthedocs.io/en/latest/quick-start.en.html>`__
+            (`obsolete
+            <https://github.com/tarantool/tarantool-python/issues/67>`__))
+
+    *   -   Testing / CI / CD
+        -   Github Actions
+        -   No (tests exists)
+        -   No (tests exists)
+        -   Travis CI and Appveyor
```

Comment:
It would be fair to say 'Travis CI (disabled) and Appveyor (only Windows)'.

Reply:
I'm not sure, it is a meaningful detail for Tarantool users.

Reply:
It allows to make a guess about code healthy and popularity of the project, so looks useful. At least if we'll fairly deliver the actual state, which is 'broken Travis CI and Appveyor for Windows' now.

Reply:
ok